### DISCUSSION TITLE
Can not view any PR, takes to 500error page

### DISCUSSION THREAD

**Discussion by DavidT1313** [community/community#134887] [Pull Requests]
### Select Topic Area

General

### Body

![image](https://github.com/user-attachments/assets/d4d08680-f523-4c32-853f-13b3acc77109)

**Comment by afollestad** [comment]
It's back now, for me

**Comment by m4ert** [comment]
Just resolved

**Comment by christophermilian** [comment]
Working again for me. Seems it was a blip.

**Comment by ghostinhershell** [comment]
Hi @DavidT1313,

Thanks for sharing and we appreciate your patience. You can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.

**Comment by thehesiod** [comment]
@ghostinhershell looks like it's happening again

**Comment by omeraha** [comment]
Same here - happening again, in PR creation flow. The status page states that everything is ok
